_shared/project/pyproject.toml

```diff
@@ -12,6 +12,8 @@ filterwarnings = [
     "error", # Fail the tests if there are any warnings.
     "ignore:^find_module\\(\\) is deprecated and slated for removal in Python 3.12; use find_spec\\(\\) instead$:DeprecationWarning:importlib",
     "ignore:^FileFinder.find_loader\\(\\) is deprecated and slated for removal in Python 3.12; use find_spec\\(\\) instead$:DeprecationWarning:importlib",
+    "ignore:^pkg_resources is deprecated as an API:DeprecationWarning:pkg_resources",
+    "ignore:^Deprecated call to .pkg_resources\\.declare_namespace\\('.*'\\).\\.:DeprecationWarning:pkg_resources",
```

Comment:
These `pkg_resources` warnings were breaking the cookiecutter's own tests since those tests use the cookiecutter to generate a project and run the project's `make sure`. This change will also add these ignores to all our projects which should fix them--they're all crashing with this warning currently.